evalml/tests/automl_tests/parallel_tests/test_automl_dask.py

```diff
@@ -283,17 +283,35 @@ def test_automl_closes_engines(engine_str, X_y_binary_cls):
 )
 @pytest.mark.parametrize("problem_type", ProblemTypes.all_problem_types)
 def test_score_pipelines_passes_X_train_y_train(
-    problem_type, engine_str, X_y_binary, X_y_regression, X_y_multi, AutoMLTestEnv
+    problem_type,
+    engine_str,
+    X_y_binary,
+    X_y_regression,
+    X_y_multi,
+    AutoMLTestEnv,
+    ts_data_binary,
+    ts_data_multi,
+    ts_data,
 ):
     if is_binary(problem_type):
```

Comment:
Now I feel like we're at a point where checking `is_binary(problem_type)` is no longer cleaner than just checking `problem_type == ProblemTypes.BINARY`, etc etc 😅